test/check-machines-create

```diff
@@ -1165,6 +1174,71 @@ vnc_password= "{vnc_passwd}"
 
             return self
 
+        def checkMiddlePageWhenCreating(self):
+            b = self.browser
+
+            buttonSelector = "#create-and-run" if self.create_and_run else "#create-and-edit"
+            b.click(buttonSelector)
+
+            if not self.create_and_run:
+                b.wait_visible(f"h1:contains(\"Creating VM {self.name}\")")
+                b.click("button:contains(\"Go to VMs list\")")
+                b.wait_visible("#virtual-machines-listing")
+            b.wait_in_text(f"#vm-{self.name}-system-state", "Creating VM")
+
+            b.click(f"#vm-{self.name}-system-name")
+            b.wait_visible(f"h1:contains(\"Creating VM {self.name}\")")
+            b.click("button:contains(\"Go to VMs list\")")
+            b.wait_visible("#virtual-machines-listing")
+            b.wait_in_text(
+                f"#vm-{self.name}-system-state",
+                "Running" if self.create_and_run else "Shut off")
+
+            b.click(f"#vm-{self.name}-system-name")
+            b.wait_visible("#vm-details")
+            b.wait_in_text("h2.vm-name", self.name)
+
+            # It seems that the difference between removing success and failure is doUsagePolling and reducer "UPDATE_VM" action.
+            # Thus, try to reload the page to trigger these event manually to see whether it could resolve the problem
```

Comment:
This feels like a bug? Or do we simply not have a good way to wait till we can delete the VM? But maybe the bigger question is, can't we re-use `runner.createTest` which removes a VM for us.

Reply:
I didn't find a way to wait untill we could detele the VM.